test/MusicStore.Test/ShoppingCartControllerTest.cs

```diff
@@ -27,6 +30,8 @@ public ShoppingCartControllerTest()
                       .AddInMemoryStore()
                       .AddDbContext<MusicStoreContext>();
 
+            services.AddMvc();
```

Comment:
I think it is because this particular action uses a custom java script to post data. 
https://github.com/aspnet/MusicStore/blob/dev/src/MusicStore/Views/ShoppingCart/Index.cshtml#L27

@Praburaj, could you clairify this?